tmt/result.py

```diff
@@ -367,3 +367,35 @@ def failures(log: Optional[str], msg_type: str = 'FAIL') -> str:
             filtered += m + '\n'
 
         return filtered or log
+
+
+def results_to_exit_code(results: list[Result]) -> int:
+    """ Map results to a tmt exit code """
+
+    from tmt.cli import TmtExitCode
+
+    stats = Result.total(results)
+
+    # Following the documentation:
+
+    # "No test results found."
+    if sum(stats.values()) == 0:
+        return TmtExitCode.NO_RESULTS_FOUND
+
+    # "Errors occured during test execution."
+    if stats[ResultOutcome.ERROR]:
+        return TmtExitCode.ERROR
+
+    # "There was a fail or warn identified, but no error."
+    if stats[ResultOutcome.FAIL] + stats[ResultOutcome.WARN]:
+        return TmtExitCode.FAIL
+
+    # "Test were executed, and all reported ``skip`` result."
```

Comment:
```suggestion
    # "Tests were executed, and all reported ``skip`` result."
```

And drop quotes?

Reply:
Actually, I would like to keep them here. The flow control `if`s would have to be commented anyway, describing the intent behind those tests, why not quote the documentation then?

Reply:
Ok.